tests/test_routes.py

```diff
@@ -34,20 +32,20 @@ def test_download_get(requests_mock, flask_test_client):
     assert page.select_one(".govuk-back-link") is None
 
 
-@pytest.mark.usefixtures("mock_get_response_json")
-def test_download_post_json(flask_test_client):
-    response = flask_test_client.post("/download", data={"file_format": "json"})
-    assert response.status_code == 200
-    assert response.mimetype == "application/json"
-    assert response.data == b'{"data": "test"}'
+# @pytest.mark.usefixtures("mock_get_response_json")
+# def test_download_post_json(flask_test_client):
+#     response = flask_test_client.post("/download", data={"file_format": "json"})
+#     assert response.status_code == 200
+#     assert response.mimetype == "application/json"
+#     assert response.data == b'{"data": "test"}'
 
 
-@pytest.mark.usefixtures("mock_get_response_xlsx")
-def test_download_post_xlsx(flask_test_client):
-    response = flask_test_client.post("/download", data={"file_format": "xlsx"})
-    assert response.status_code == 200
-    assert response.mimetype == "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
-    assert response.data == b"xlsx data"
+# @pytest.mark.usefixtures("mock_get_response_xlsx")
+# def test_download_post_xlsx(flask_test_client):
```

Comment:
Could we use request-mock here and check the correct call is made to the data-store?

Reply:
I've added unit tests to test_routes.py, but I'm not sure if that's enough.